### ISSUE TITLE
I have a question.

### ISSUE THREAD

**Issue by jinguman** [huaxiuyao/C-Mixup#1]
Hello.
First of all, thank you for source.

Is there a reason why scheduler.step() is missing in the train() function of algorithm.py when "is_mixup" is True?
